app/components/press-release/press-release-read-only.hbs

```diff
@@ -0,0 +1,98 @@
+<div class="auk-u-mt-8 auk-container auk-container--center">
+  <div class="auk-u-mt-8">
+        <div class="auk-o-flex auk-o-flex--vertical">
+      <div class="auk-o-grid auk-u-mb-4">
+        <Auk::Label for="title" class="auk-o-grid-col-3">
+          Titel
+        </Auk::Label>
+        <h2 class="auk-h2 auk-o-grid-col-9">{{@pressRelease.title}}</h2>
+      </div>
+      <div class="auk-o-grid auk-u-mb-4">
+        <Auk::Label for="abstract" class="auk-o-grid-col-3">
+          Samenvatting
+        </Auk::Label>
+        <p class="auk-o-grid-col-9">{{@pressRelease.abstract}}</p>
+      </div>
+      <div class="auk-o-grid auk-u-mb-4">
+        <Auk::Label for="message" class="auk-o-grid-col-3">
+          Inhoud
+        </Auk::Label>
+        <p class="auk-o-grid-col-9">{{@pressRelease.htmlContent}}</p>
+      </div>
+      <div class="auk-o-grid auk-u-mb-4">
+        <Auk::Label class="auk-o-grid-col-3">
+          Bron
+        </Auk::Label>
+        <ul class="auk-o-grid-col-9">
+          {{#each @pressRelease.sources as |source|}}
+            <li>
+              {{source.fullName}}
```

Comment:
Bij een bron willen we meer informatie tonen. Eventueel dezelfde tabel als in het formulier, maar dan zonder de delete-buttons.